### DISCUSSION TITLE
NoReverseMatch when adding tag to circuits [3.2.0-beta1]

### DISCUSSION THREAD

**Discussion by nunesvn** [netbox-community/netbox#8776] [General]
Hello, I'm having this problem on 3.2.0-beta1 and I'm not sure where to report, as is a beta version.

**Environment**
Python version: 3.8.10
NetBox version: 3.2.0-beta1
(https://beta-demo.netbox.dev and locally)

**Steps to Reproduce**
1. Create a Tag "tag1"
2. Create a Circuit "circuit1"
3. Add tag1 to circuit1
4. Go to tag1 details page and get a NoReverseMatch (https://beta-demo.netbox.dev/extras/tags/<tag number>/

**Expected Behavior**
Details page of tag1

**Observer Behavior**
```
<class 'django.urls.exceptions.NoReverseMatch'>

Reverse for 'taggeditem_edit' not found. 'taggeditem_edit' is not a valid view function or pattern name.
```

**Comment by 3bruno** [comment]
Hi... 3.2.0-beta2 have the same issue.

**Comment by jeremystretch** [comment]
> I'm not sure where to report, as is a beta version.

Should be a bug report, same as with the stable release. I've opened #8869 for this. Thanks!
